tools/IntegrationsJsonGenerator/IntegrationsJsonGenerator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net7.0</TargetFrameworks>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
I would revert this file. Parents props file is using `TargetFrameworks`. Potentially it can introduce some unexpected behabiout.

Reply:
When both `TargetFrameworks` and `TargetFramework` are present in a `.csproj` the first one is ignored. This is well-defined behavior, but, if it is confusing we can either add a comment here or add the `--framework` platform setting directly on the build source. Any preference?

Reply:
Lets keep it as it is.
Thanks for explanation.

Reply:
It makes more sense to add the comment: it is an executable, and we want to build to a single runtime, so running with the latest available avoids forcing the code to compile to .NET Framework too. The app code can evolve with the latest runtime, BCL, without concerns.